DonjinKrawler-Console/src/main/java/donjinkrawler/CommandParser.java

```diff
@@ -0,0 +1,40 @@
+package donjinkrawler;
+
+import donjinkrawler.interpreter.*;
+
+public class CommandParser {
```

Comment:
Reiktu padaryti final. 

[CHECKLIST](https://dzone.com/articles/java-code-review-checklist):
`Make class final if not being used for inheritance.`